backend/deepchecks_monitoring/monitoring_utils.py

```diff
@@ -78,6 +78,23 @@ def __repr__(self) -> str:
         """Return string representation."""
         return self.__str__()
 
+    # TODO: replace current __str__ with this
+    def stringify(self) -> str:
```

Comment:
i think a dict will be more elegant

Reply:
aaaa, do not know) probably it is just a question of taste
from one point, yeap, it will reduce a few lines, but from another point, 
the Python interpreter will need to execute more instructions